### ISSUE TITLE
"Collapse tags at this level" only works for tags in the current view of the tag pane

### ISSUE THREAD

**Issue by SkepticMystic** [pjeby/tag-wrangler#2]
When using the command "Collapse tags at this level" to fold tags in the tag pane, only tags in the current view are folded, instead of _all_ tags.

For example, if the full tag list is:

```
Note
    NotStarted
    InProgress
Book
    Read
    NotRead
Movie
    Watched
    NotWatched
```

But the current view only shows:

```
Note
    NotStarted
    InProgress
Book
    Read
    NotRead
```

Then using the command "Collapse tags at this level" on the `Book` tag will only collapse the `Note` and `Book` tags, not the `Movie` tag:

```
Note >
Book >
Movie
    Watched
    NotWatched
```

**Comment by pjeby** [comment]
Fixed in 5393d25, released as 0.2.4.  I tested with a relatively small tag set, so let me know if the problem persists after updating to 0.2.4.

**Comment by SkepticMystic** [comment]
That works, thanks for the quick update 👍
